### PR TITLE
Feature/40630

### PR DESCRIPTION
**Pull request by Aekal**
Internal 40630

### CODE REVIEW

File: components/04-views/compare/_compare.scss [hunk 1]
```diff
@@ -0,0 +1,38 @@
+$compare__border          : 1px solid $gray-lighter !default;
+$compare__margin-bottom   : $spacer--medium !default;
+$compare__label-background: #f7f7f9 !default;
```
Comment (bartdominiak) [inline components/04-views/compare/_compare.scss]: use $gray-lightest

File: components/04-views/compare/_compare.scss [hunk 2]
```diff
@@ -1,9 +1,24 @@
-$compare__border          : 1px solid $gray-lighter !default;
-$compare__margin-bottom   : $spacer--medium !default;
-$compare__label-background: #f7f7f9 !default;
-$compare__label-padding   : $spacer--medium !default;
-$compare__cell-padding    : $spacer--medium !default;
-$compare__cell-min-width  : 160px !default;
+$compare__border                   : 1px solid $gray-lighter !default;
+$compare__margin-bottom            : $spacer--medium !default;
+$compare__label-background         : #f7f7f9 !default;
```
Comment (bartdominiak) [inline components/04-views/compare/_compare.scss]: use `$gray-lightest`

File: components/04-views/compare/_compare.scss (hunk 1 above)
Comment (Aekal) [reply components/04-views/compare/_compare.scss]: fixed

File: components/04-views/compare/_compare.scss (hunk 2 above)
Comment (Aekal) [reply components/04-views/compare/_compare.scss]: fixed

**Pull request merged**